### DISCUSSION TITLE
How do images work through SC?

### DISCUSSION THREAD

**Discussion by vdruts** [brianpetro/obsidian-smart-connections#847] [General]
Can I put images into a .md and will the models see them appropriately? How do images work?

**Comment by brianpetro** [comment]
In v2.3, images are processed to extract text-from and description-of the image. The extracted content is used to surface relevant images in the Smart Connections view.

Currently, links are not rendered when the chat model "sees" a note. So it will still just see the markdown link.

"Side-loading" content, for example loading the content of links adjacent to the primary note, is something I'm working on for the Notes v2 GPT via the Smart Connect app. If that works out, I imagine it will make it's way into the Smart Chat.

As far as "side-loading" for connections, there are some other techniques that may provide equivalent outcomes (i.e., properly adjusting the "connection score" based on the links) that I plan to try before "side-loading" in the embedding input 🌴
